cloudformation/membership-attribute-service.yaml

```diff
@@ -178,6 +178,8 @@ Resources:
         Ref: LaunchConfig
       MinSize:
         Ref: MinInstances
+      DesiredCapacity:
```

Comment:
I would not explicitly set desired in this template, see the note at: https://riffraff.gutools.co.uk/docs/magenta-lib/types#cloudformation